client/reconfiguration/src/default_handlers.cpp

```diff
@@ -71,20 +70,38 @@ bool ClientTlsKeyExchangeHandler::execute(const State& state, WriteState&) {
   if (master_key.empty()) master_key = psm_.decryptFile(master_key_path_).value_or(std::string());
   if (master_key.empty()) LOG_FATAL(getLogger(), "unable to read the node master key");
   auto root = fs::path(cert_folder_);
-  for (const auto& c : bft_clients_) {
-    fs::path root_path = root / std::to_string(c);
-    bool use_unified_certs = bftEngine::ReplicaConfig::instance().useUnifiedCertificates;
-    fs::path pkey_path =
-        (use_unified_certs) ? root_path / ("pk.pem" + suffix) : root_path / "client" / ("pk.pem" + suffix);
-    fs::path cert_path = (use_unified_certs) ? root_path / "node.cert" : root_path / "client" / "client.cert";
+  if (use_unified_certificates_) {
+    LOG_INFO(getLogger(), "Using unified certificates");
+    fs::path root_path = root / std::to_string(clientservice_pid_);
```

Comment:
Nit: use 'const fs::path' here and below

Reply:
done